dev/pr-checks.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Perhaps we could update the CI to run this script, to prevent it from drifting?

Reply:
I've renamed the script to "format-code" as that is what it's really doing.